jekyll/_cci2/configuration-reference.md

```diff
@@ -558,6 +558,7 @@ Key | Required | Type | Description
 ----|-----------|------|------------
 paths | Y | List | List of directories which should be added to the cache
 key | Y | String | Unique identifier for this cache
+name | N | String | Title of the step to be shown in the CircleCI UI (default: `save_cache`)
```

Comment:
I think the default appears as Restoring Cache and Saving Cache in the UI, so suggest to change defaults to those.